libafl_qemu/src/elf.rs

```diff
@@ -67,6 +67,32 @@ impl<'a> EasyElf<'a> {
         None
     }
 
+    #[must_use]
+    pub fn get_section(&self, name: &str, load_addr: GuestAddr) -> Option<Range<GuestAddr>> {
+        for section in &self.elf.section_headers {
+            if let Some(section_name) = self.elf.shdr_strtab.get_at(section.sh_name) {
+                println!(
```

Comment:
This probably also wants to use `logger::something`

Reply:
Fixed